front/src/layouts/footer/footer.css

```diff
@@ -0,0 +1,64 @@
+div.body {  
+  width: 1440px;
+}
+
+div.main, div.lowerPart {  
+  display: flex;
+  flex-direction: row;                        
+  align-items: center;
+  justify-content: space-between; 
+}
+
+div.main {
+  height: 125px;
+}
+
+div.logo {
+  margin: 5px 0px 10px 80px;
+  padding: 5px 0px 10px;   
+}
+
+div.stores {
+  margin-left: 80px;   
+}
+
+div.lowerPart {
+  height: 57px; 
+}
+
+div.lowerPart > span {
```

Comment:
it is too specific and unflexible, i am not sure that you need class lower-part at all

Reply:
so, the bottom of the footer is a different color